spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/quantized/SpatialConvolution.scala

```diff
@@ -47,8 +47,18 @@ private[bigdl] class SpatialConvolution[T: ClassTag](
   require(nInputPlane % nGroup == 0, "Number of input channels should be multiples of group.")
   require(nOutputPlane % nGroup == 0, "Number of output channels should be multiples of group.")
 
+  private val data: QuantizedTensor[T] = QuantizedDummyTensor[T]()
```

Comment:
@i8run Please help take a look at the change here. The original code looks like have some bug. As the param is used before it's inited

Reply:
Yes. `quantFormat` should be initialized first.